src/wp-includes/block-template-utils.php

```diff
@@ -290,20 +290,41 @@ function _get_block_template_file( $template_type, $slug ) {
  * Retrieves the template files from the theme.
  *
  * @since 5.9.0
+ * @since 6.3.0 Added the `$query` parameter.
  * @access private
  *
  * @param string $template_type 'wp_template' or 'wp_template_part'.
- * @return array Template.
+ * @param array  $query {
+ *     Arguments to retrieve templates. Optional, empty by default.
+ *
+ *     @type array  $slug__in      List of slugs to include.
+ *     @type array  $slug__not_in  List of slugs to skip.
+ *     @type string $area          A 'wp_template_part_area' taxonomy value to filter by (for wp_template_part template type only).
+ *     @type string $post_type     Post type to get the templates for.
```

Comment:
```suggestion
 *     @type array  $slug__in     List of slugs to include.
 *     @type array  $slug__not_in List of slugs to skip.
 *     @type string $area         A 'wp_template_part_area' taxonomy value to filter by (for wp_template_part template type only).
 *     @type string $post_type    Post type to get the templates for.
```

Nit-pick, Remove extra space.